tracer/test/benchmarks/Benchmarks.Trace/OpenTelemetryManualBenchmark.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Specialized;
+using System.Diagnostics;
+using BenchmarkDotNet.Attributes;
+using Datadog.Trace;
+using Datadog.Trace.Activity.DuckTypes;
+using Datadog.Trace.Configuration;
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Resources;
+using ActivityListener = System.Diagnostics.ActivityListener;
+using ActivitySamplingResult = System.Diagnostics.ActivitySamplingResult;
+using ActivitySource = System.Diagnostics.ActivitySource;
+
+
+namespace Benchmarks.Trace;
+
+[MemoryDiagnoser]
+[BenchmarkAgent5]
+public class OpenTelemetryManualBenchmark
+{
+    internal static ActivitySource ActivitySource { get; set; }
+    internal static Datadog.Trace.Tracer DatadogTracer { get; set; }
+    internal static OpenTelemetry.Trace.Tracer OpenTelemetryTracer { get; set; }
+    static OpenTelemetryManualBenchmark()
+    {
+        ActivitySource = new ActivitySource("ActivityBenchmark");
+
+        var activityListener = new ActivityListener
+        {
+            ShouldListenTo = _ => true,
+            Sample = (ref ActivityCreationOptions<ActivityContext> options) => ActivitySamplingResult.AllData
+        };
+
+        ActivitySource.AddActivityListener(activityListener);
+
+        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                                      .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("dd-trace-dotnet"))
+                                      .AddSource("ActivityBenchmark")
+                                      .AddConsoleExporter()
+                                      .Build();
+
+        OpenTelemetryTracer = tracerProvider.GetTracer("dd-trace-dotnet");
+        var environmentVars = new NameValueCollection();
+        environmentVars.Add("DD_TRACE_OTEL_ENABLED", "false");
+
+        var settings = new TracerSettings(new NameValueConfigurationSource(environmentVars)) { StartupDiagnosticLogEnabled = false };
+
+        Datadog.Trace.Tracer.UnsafeSetTracerInstance(new Datadog.Trace.Tracer(settings, new DummyAgentWriter(), null, null, null));
+        Datadog.Trace.ClrProfiler.Instrumentation.Initialize(); // enabled, but we don't have OTEL enabled
```

Comment:
I'm not sure I understand the point of this benchmark 🤔 Is it meant to be for comparing with the "auto" benchmark? Maybe it would be better to call it "uninstrumented" if so. And also, if so, I don't think we need to configure any of the tracer stuff do we?

Reply:
Yeah that was the idea - configure the tracer the same just with the OpenTelemetry disabled instead of enabled

I guess it'd make sense to remove all of the tracer configuration stuff though for this

Reply:
> I guess it'd make sense to remove all of the tracer configuration stuff though for this

Yeah, doesn't seem worth setting up seeing as they're not being used, and also remember `Datadog.Trace.ClrProfiler.Instrumentation.Initialize()` can only be called once per process, which means one or other of your benchmarks is "broken" depending on the order they run in 😬

Reply:
I opted to remove this file for the time being as probably just benchmarking `Activity` / `OpenTelemetry` without the tracer doesn't make sense.